src/components/leaderboard/leaderboardTypography.module.css

```
@@ -0,0 +1,10 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700;800&display=swap");
+@import url("https://fonts.cdnfonts.com/css/goldplay");
+
+.goldplay {
+  font-family: "Goldplay", "Open Sans", sans-serif;
```

Stylelint (17.7.0)

[error] Expected no quotes around "Goldplay" (font-family-name-quotes)

(font-family-name-quotes)